ecoscope/io/earthranger.py

```diff
@@ -350,8 +379,8 @@ def get_subject_observations(
         Get observations for each listed subject and create a `Relocations` object.
         Parameters
         ----------
-        subject_ids : str or list[str]
-            List of subject UUIDs
+        subjects : str or list[str] or pd.DataFrame
```

Comment:
Isn't subject_ids more precise? I think we should always use either subject_ids or subject_names if possible to help prevent confusion.

Reply:
I thought it might have been a bit confusing that one could pass a DataFrame of subjects to a param called subject_ids. But happy either way